test/e2e/menuPage.cy.js

```diff
@@ -1,8 +1,13 @@
+import {getData} from './commands'
+
 describe('Menu Page', () => {
-  const pageTitle = 'Adapt Version 5';
 
-  it(`should have the title ${pageTitle}`, () => {
+  beforeEach(() => {
```

Comment:
```suggestion
  before(() => {
```

I think we could reduce calls here by just making this a `before` for one run instead of on each test scenario

Reply:
You'll want clean data for each test. If any test modifies the data, the following test will otherwise have to be aware of changes in the preceding tests. The best practise would be to load the data fresh for each test or at least clone it each time.

Reply:
Just to add to this, Cypress will also clear aliases after each test to avoid data manipulation issues.